x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_edit.tsx

```diff
@@ -73,10 +81,15 @@ const cloneAndMigrateRule = (initialRule: Rule) => {
             initialRule.notifyWhen === RuleNotifyWhen.THROTTLE ? initialRule.throttle! : null,
         }
       : { summary: false, notifyWhen: RuleNotifyWhen.THROTTLE, throttle: initialRule.throttle! };
-    clonedRule.actions = clonedRule.actions.map((action) => ({
-      ...action,
-      frequency,
-    }));
+    clonedRule.actions = clonedRule.actions.map((action: SanitizedRuleAction) => {
+      if (actionHasDefinedGroup(action as SanitizedRuleAction)) {
+        return {
+          ...action,
+          frequency,
+        } as SanitizedRuleAction;
+      }
+      return action as SanitizedRuleAction;
```

Comment:
Let's add some jest tests to this as well, also curious to know why we have to cast the types so often, maybe worth trying to fix them, unless we're migrating types like we were the the http versioning stuff